### PR TITLE
1464129: Put guestId param in consumer json for virtWho instances

### PR DESCRIPTION
**Pull request by wottop**

### CODE REVIEW

**Comment by wottop** [comment]
[1458674: Force Candlepin JSON generation for hypervisor checkin to include empty list for guests](https://trello.com/c/bsCziudd/463-1458674-force-candlepin-json-generation-for-hypervisor-checkin-to-include-empty-list-for-guests)

**Comment by mstead** [comment]
retest this please

**Comment by wottop** [comment]
https://bugzilla.redhat.com/show_bug.cgi?id=1464129

File: server/src/main/java/org/candlepin/model/Consumer.java [hunk 1]
```diff
@@ -791,4 +791,14 @@ public void setRecipientOwnerKey(String recipientOwnerKey) {
         this.recipientOwnerKey = recipientOwnerKey;
     }
 
+    /**
+     * This will put in the property so that the virtWho instances won't error
+     *
+     * @return List always empty
+     */
+    @JsonProperty("guestIds")
+    public List getEmptyGuestIdArray() {
```
Comment (Ceiu) [inline server/src/main/java/org/candlepin/model/Consumer.java]: Are we ever going to use this property going forward?

**Comment by Ceiu** [comment]
ACK

**Pull request merged**